management-interface.go

```diff
@@ -35,6 +35,9 @@ import (
 // Using a packr box means the html files are bundled up in the binary application.
 var templateBox = packr.NewBox("./html")
 
+// Using a packr box means the sound files are bundled up in the binary application.
+var soundsBox = packr.NewBox("./sounds")
```

Comment:
it seems like we're going to package a test sound with the executable so this isn't necessary any more